packages/pipelines/src/commands/reviewapps/disable.ts

```diff
@@ -14,11 +14,23 @@ export default class ReviewappsDisable extends Command {
     pipeline: flags.string({char: 'p', description: 'name of pipeline', required: true}),
     autodeploy: flags.boolean({description: 'disable autodeployments', required: false}),
     autodestroy: flags.boolean({description: 'disable automatically destroying review apps', required: false}),
+    app: flags.string({description: 'parent app used by review apps', required: false}),
```

Comment:
You can use the `flags.app` helper here, `app: flags.app({description: 'parent app used by review apps', required: false})`. This flag partners with `flags.remote`.